lib/js/atoms/Modal/Modal.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+	<div class="m-modal" @click.stop="$emit('close-modal')">
```

Comment:
Why `m-modal`? Is this because of https://github.com/jgthms/bulma/blob/243880f004343a41b2dc95003d2b1530900de118/sass/components/modal.sass#L71? If so, I think it's time to figure out a better way to address this issue, together with https://github.com/bethinkpl/design-system/pull/71/files#r745467973

Reply:
`ds-modal` + `DsModal`?
https://github.com/bethinkpl/design-system/blob/3455d9d8fc6fbcc186902ef403b240daa8eba80b/lib/js/atoms/SurveyQuestion/SurveyQuestionOpenEnded.vue#L6

Reply:
Addressed in https://github.com/bethinkpl/design-system/pull/71#discussion_r745874430